packages/devtools_app/lib/src/screens/inspector/inspector_tree_controller.dart

```diff
@@ -761,6 +759,9 @@ class _InspectorTreeState extends State<InspectorTree>
 
   late DebuggerController _debuggerController;
 
+  /// When autoscrolling, the number of rows to pad the target location with.
+  final _scrollToTargetPaddingCount = 3;
```

Comment:
Add Row to this constant name as it is talking about a number of rows not a number of px.